FlowCrypt/Controllers/SideMenu/Menu/MyMenuViewController.swift

```diff
@@ -115,6 +185,7 @@ final class MyMenuViewController: ASDKViewController<ASDisplayNode> {
         tableNode.reloadData()
     }
 
+    // TODO: - ANTON - Handle error
```

Comment:
Various todos in this file - thank you for marking them clearly :+1:

Reply:
will make them more readable next time)